src/com_tjnotifications/admin/models/notifications.php

```diff
@@ -119,20 +119,21 @@ protected function getListQuery()
 	 *
 	 * @param   String  $client  An optional array of data for the form to interogate.
 	 * @param   String  $key     True if the form is to load its own data (default case), false if not.
+	 * @param   String  $language  A string of data for the form to interogate.
 	 *
 	 * @return  JForm  A JForm object on success, false on failure
 	 *
 	 * @since    1.6
 	 */
-	public function getTemplate($client, $key)
+	public function getTemplate($client, $key, $language)
 	{
 		$object = clone $this;
 
 		$this->setState('filter.key', $key);
 		$this->setState('filter.client', $client);
```

Comment:
add one more setState for language, and use  `$templates = $this->getItems();`	instead of creating new function getSpecificTemplate